Cargo.toml

```diff
@@ -20,5 +20,6 @@ optional_struct_macro = { version = "0.5.2", path = "optional_struct_macro" }
 serde = { version = "1.0.193", features = ["derive"], default-features = false }
 
 [dev-dependencies]
+clap = { version = "4.5.16", features = ["derive", "std"] }
```

Comment:
I don't really like adding an extra dependency just for this short example.

I will let that slide for now, but if I improve the crate to better support attributes (i.e. something generic rather than our serde-specific stuff), I would probably remove that as well.